Parsely/wp-parsely
# Remove phpunit-integration.xml.dist from exported plugin files

## Description


Add a wildcard to the phpunit line in the .gitattributes file

## Motivation and Context



This is a development file and does not belong in a published plugin archive (be it zip or dir). We already ignore the file called `phpunit.xml.dist` -- this change adds a wildcard such that it will ignore any similarly named files in the future.

## How Has This Been Tested?




* Check out this branch
* Run ./bin/export-plugin.sh ~/Downloads/wp-parsely-dev DIR
* `ls ~/Downloads/wp-parsely-dev` -- should be a properly formed dir of "exported" plugin files without either of the `phpunit*.xml.dist` files

## Screenshots (if appropriate)
